packages/viewer/src/scenes/camera.ts

```diff
@@ -282,20 +375,44 @@ export class Camera implements FrameCamera.FrameCamera {
     return { ...this.data.position };
   }
 
+  /**
+   * The position vector for the camera when fit to the visible bounding box,
+   * in world space coordinates.
+   */
+  public get positionFitAll(): Vector3.Vector3 {
+    return { ...this.fitToBoundingBox(this.boundingBox).position };
+  }
```

Comment:
These fit all getters seem unnecessary to me. With this API, you start to go down the path of having getters for each of the camera fields for each of the different states a camera can be in. Just have the caller call `fitToBoundingBox` once and they should have access to the attributes of the fit camera.